kernel/src/net/socket/netlink/kobject_uevent/message/uevent.rs

```diff
@@ -0,0 +1,176 @@
+// SPDX-License-Identifier: MPL-2.0
+
+use alloc::format;
+use core::{
+    str::FromStr,
+    sync::atomic::{AtomicU64, Ordering},
+};
+
+use super::syn_uevent::{SyntheticUevent, Uuid};
+use crate::prelude::*;
+
+/// `SysObj` action type.
+///
+/// Reference: <https://elixir.bootlin.com/linux/v6.14/source/include/linux/kobject.h#L53>.
+#[derive(Debug, Clone, Copy, PartialEq, Eq, TryFromInt)]
+#[repr(u8)]
+pub enum SysObjAction {
```

Comment:
Types under `uevent/` do not have any users, so it's hard to say what's the desired visibility for them. Maybe you have an idea?

Reply:
I haven't determined where to put these files. They may be moved to `comps/systree` in the future.

If they are put under `net/socket/netlink`, I guess most APIs need to be public. `SysNode` needs to generate uevent and multicast uevent message when some events happen.

Reply:
Would you mind giving _all the types and methods_ under `kernel/src/net/socket/netlink/kobject_uevent/message/` the _most_ restricted visibility marker?

I should have gone through all `pub` items added in this PR. These types and methods are the only ones I cannot justify.

We can, of course, relax the visibility in the future. But for now, let's make it clear they're dead code.